### PR TITLE
Add a sample exhibiting custom Razor Page handler conventions

### PR DESCRIPTION
**Pull request by pranavkm**
Fixes https://github.com/aspnet/Mvc/issues/6065

### CODE REVIEW

File: samples/Mvc.CustomRazorPageHandlers/CustomHandlerPageApplicationModelProvider.cs [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+        }
+        else if (
+          string.Equals(httpMethod, "DELETE", StringComparison.OrdinalIgnoreCase) || 
+          string.Equals(httpMethod, "PUT", StringComparison.OrdinalIgnoreCase))
+        {
+          // e.g. DeleteUser
+          // HttpMethod = "POST", Handler = "DeleteUser"
+          httpMethod = "POST";
+          handler = methodName;
+        }
```
Comment (ajaybhargavb) [inline samples/Mvc.CustomRazorPageHandlers/CustomHandlerPageApplicationModelProvider.cs]: Shouldn't this have
```c#
else
{
    return false;
}
```
?

File: samples/Mvc.DynamicCorsPolicyProvider/Mvc.DynamicCorsPolicyProvider.sln [hunk 2]
```diff
@@ -1,22 +0,0 @@
-﻿
```
Comment (ajaybhargavb) [inline samples/Mvc.DynamicCorsPolicyProvider/Mvc.DynamicCorsPolicyProvider.sln]: Intentional?

File: samples/Mvc.CustomRazorPageHandlers/CustomHandlerPageApplicationModelProvider.cs (hunk 1 above)
Comment (pranavkm) [reply samples/Mvc.CustomRazorPageHandlers/CustomHandlerPageApplicationModelProvider.cs]: Yup.

File: samples/Mvc.DynamicCorsPolicyProvider/Mvc.DynamicCorsPolicyProvider.sln (hunk 2 above)
Comment (pranavkm) [reply samples/Mvc.DynamicCorsPolicyProvider/Mvc.DynamicCorsPolicyProvider.sln]: Yeah, pretty sure we don't have xproj anymore or require a sln file with this one project.

**Comment by pranavkm** [comment]
🆙 📅

**Pull request merged**